python/instrumentation/openinference-instrumentation-pydanticai/src/openinference/instrumentation/pydanticai/__init__.py

```diff
@@ -0,0 +1,9 @@
+from openinference.instrumentation.pydanticai.span_exporter import OpenInferenceSpanExporter
+from openinference.instrumentation.pydanticai.utils import is_openinference_span
+from openinference.instrumentation.pydanticai.version import __version__
+
+__all__ = [
+    "OpenInferenceSpanExporter",
+    "is_openinference_span",
+    "__version__",
+]
```

Comment:
```suggestion
from openinference.instrumentation.pydanticai.span_exporter import OpenInferenceSpanExporter
from openinference.instrumentation.pydanticai.version import __version__

__all__ = [
    "OpenInferenceSpanExporter",
    "__version__",
]
```

This looks like a helper function we are using in tests only rather than something we want to expose to end users?

Reply:
Yeah this was kind of copying the vercel js implementation. I assumed it's because you end up with some spans that don't map to openinference ones (something I saw as well in pydantic). I was thinking it is a convenience function for a user to block those from getting to phoenix, but probably need more context (maybe @Parker-Stafford can chime in) on what that was for in vercel